boards/pic32-wifire/pic32_config_settings.c

```diff
@@ -0,0 +1,403 @@
+/*
```

Comment:
I am not really familiar with the MIPS architecture: looking at this file, it seems like it contains many configuration options that I would expect to be done by their connected modules/periph drivers. Do all of these options need to be set during compile time? And are they read only later on? In the current form these options seem to be very hard to maintain/to configure for others...